src/zulip.rs

```diff
@@ -339,9 +356,32 @@ async fn workqueue_commands(
                 }
             )
         }
-        "help" => r"work show => show your assigned PRs
-work set-pr-limit <number>|unlimited => set the maximum number of PRs you can be assigned to"
-            .to_string(),
+        "set-rotation-mode" => {
+            let rotation_mode = match words.next() {
+                Some(value) => {
+                    if words.next().is_some() {
+                        anyhow::bail!("Too many parameters.");
+                    }
+                    match value {
+                        "on" => RotationMode::OnRotation,
+                        "off" => RotationMode::OffRotation,
+                        _ => anyhow::bail!("Unknown rotation mode {value}. Use `on` or `off`."),
+                    }
+                }
+                None => anyhow::bail!("Missing parameter."),
+            };
+            upsert_review_prefs(
+                &db_client,
+                user,
+                review_prefs.and_then(|p| p.max_assigned_prs.map(|v| v as u32)),
+                rotation_mode,
+            )
+            .await
+            .context("Error occurred while setting review preferences.")?;
+            tracing::info!("Setting rotation mode `{gh_username}` to {rotation_mode:?}");
+            format!("Rotation mode set to {rotation_mode:?}")
+        }
+        "help" => WORKQUEUE_HELP.to_string(),
         _ => anyhow::bail!("Invalid subcommand."),
```

Comment:
nit: we could/should mention the `help` command when we encounter an invalid subcommand here

Reply:
That's already done on the higher level, if anything fails here, help is automatically printed.

Reply:
Great.